base/sparse/sparsevector.jl

```diff
@@ -636,8 +636,8 @@ function showarray(io::IO, x::AbstractSparseVector;
 end
 
 function summary(x::AbstractSparseVector)
-    string("Sparse vector of length ", length(x), ", with ", length(nonzeros(x)),
-           " ",  eltype(x), " nonzero entries:")
+    string(length(x), "-element sparse vector with ", length(nonzeros(x)),
```

Comment:
I've added "nonzero entries" to both matrix and vector. (I'd prefer we say "stored entries", but this would have to be changed in the whole code base, including renaming functions.)